doc/modules/model_evaluation.rst

```diff
@@ -1033,6 +1033,47 @@ set [0,1] has an error: ::
     for an example of zero one loss usage to perform recursive feature
     elimination with cross-validation.
 
+.. _brier_score_loss:
+
+Brier score loss
+----------------
+
+The :func:`brier_score_loss` function returns a score of the mean square difference
+between the actual outcome and the predicted probability of the possible outcome.
+The actual outcome has to be 1 or 0 (true or false), while the predicted probability
+of the actual outcome happens can be value between 0 and 1. The brier score loss is
+also between 0 to 1 and the lower the score (the mean square difference is smaller),
+the more accurate the prediction is.
+
```

Comment:
This paragraph need to give some hint on the "why" of this score function. For instance:

The Brier score was proposed by Glenn W. Brier in 1950. It can be thought of as either a measure of the "calibration" of a set of probabilistic predictions.

And please also add a reference to the original paper:

http://docs.lib.noaa.gov/rescue/mwr/078/mwr-078-01-0001.pdf